### PR TITLE
Redis

### PR DESCRIPTION
**Pull request by QubitPi**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: elide-example/elide-hibernate5-redis-example/src/main/java/com/yahoo/elide/example/persistence/ElideResourceConfig.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ * Example application for resource config
+ */
+public class ElideResourceConfig extends ResourceConfig {
+    public ElideResourceConfig() {
+        register(new AbstractBinder() {
+            @Override
+            protected void configure() {
+                JsonApiEndpoint.DefaultOpaqueUserFunction noUserFn = v -> null;
```
Comment (DeathByTape) [inline elide-example/elide-hibernate5-redis-example/src/main/java/com/yahoo/elide/example/persistence/ElideResourceConfig.java]: Not really a comment for change here, but we really should provide an example for how to properly interleave users and user auth.

**Comment by DeathByTape** [comment]
The code looks good to me, but I would just suggest we add a redis-specific integration test to the hibernate5 datastore.

**Comment by DeathByTape** [comment]
Is this branch now stale @QubitPi? (i.e. should we close it?)

**Comment by QubitPi** [comment]
@aklish would like to add Redis after Elide 3.0 is released. I'm OK with closing this one, but I will re-do this PR anyway if we still plan to add Redis

**Pull request closed without merging**